go.mod

```diff
@@ -1,9 +1,13 @@
 module github.com/pmenglund/high-frequency-lambda
 
-go 1.13
+go 1.20
```

Comment:
go1.21 is out

Reply:
Let's not force everyone to upgrade just in case.